openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -541,6 +559,7 @@ def verify_get_own_information(queries):
             assert 'Learn a lot' == data['goals']
             assert self.user.email == data['email']
             assert self.user.id == data['id']
+            assert data['activation_key'] is None
```

Comment:
remove this assertion. It will cause confusion for activation_key to be none in success cases(though it is possible). Since there is already a separate unit test for activation key checks, no need to add a rather confusion-causing assertion

Reply:
In order to remove this, it'll have to be deleted from the dict before assertion and there are multiple full response test functions.

Reply:
nvm, got it. Just removing these assertions.